test/functional/apps/discover/_field_data_with_fields_api.ts

```diff
@@ -92,12 +91,14 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           hash.replace('columns:!()', 'columns:!(relatedContent)'),
           { useActualUrl: true }
         );
+
+        await PageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
Does this mean that we need to add this to all places where navigation in `Discover` occurs